src/ontology/reports/sync-synonym/review-qc-duplicate-exact-synonym-no-abbrev.tsv

```diff
@@ -1807,8 +1807,8 @@ cancer of nasal cavity	MONDO:0003212	DOID:4931	updated
 cancer of nasal cavity	MONDO:0003212	NCIT:C9336	updated		
 cancer of nasopharynx	MONDO:0015459	NCIT:C3871	updated		
 cancer of nasopharynx	MONDO:0021315	MONDO:patterns/cancer	confirmed		
-cancer of oesophagus	MONDO:0007576		confirmed	http://purl.obolibrary.org/obo/OMO_0003005	
-cancer of oesophagus	MONDO:0019086	DOID:1107	updated	http://purl.obolibrary.org/obo/OMO_0003005	
+cancer of oesophagus	MONDO:0007576		confirmed	http://purl.obolibrary.org/obo/OMO_0003005|http://purl.obolibrary.org/obo/OMO_0003005	
```

Comment:
Another case I need to fix of duplicate synonym type showing up here. Or we can just remove the synonym_type column from this review file; if it's not that helpful, we don't need it.